go/chat/utils/utils.go

```diff
@@ -944,6 +944,17 @@ func CreateTopicNameState(cmp chat1.ConversationIDMessageIDPairs) (chat1.TopicNa
 	return h.Sum(nil), nil
 }
 
+func GetConvLatestCtime(rc types.RemoteConversation) gregor1.Time {
```

Comment:
From other PR, I don't really like this name. Ctime of what? Thing it should be explicit about it being the last send time of the current user.